pkg/provision/workspace/ssh.go

```diff
@@ -0,0 +1,94 @@
+//
+// Copyright (c) 2019-2024 Red Hat, Inc.
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+package workspace
+
+import (
+	_ "embed"
```

Comment:
Is there a reason why the blank identifier `_` is being used? Couldn't we just have 
```go
import (
	"embed"
...
)
```

I've never used the embed directive, but [examples of it](https://gobyexample.com/embed-directive) don't seem to be using the blank identifier.

Reply:
Importing the `embed` explicitly causes an error: `pkg/provision/workspace/ssh.go:19:2: "embed" imported and not used`